webhooks/ssp_webhook.go

```diff
@@ -189,6 +200,22 @@ func newSspValidator(clt client.Client) *sspValidator {
 
 func getSsp(obj runtime.Object) (*sspv1beta2.SSP, error) {
 	switch sspObj := obj.(type) {
+	case *sspv1beta1.SSP:
```

Comment:
If it is not served, is this actually possible?

Reply:
removed